helm/prometheus-rules/templates/platform/atlas/alerting-rules/fluentbit.rules.yml

```diff
@@ -18,7 +18,7 @@ spec:
         __dashboardUid__: fluentbit
         dashboardQueryParams: "orgId=2"
       # If we have some failures sending data, raise an alert
-      expr: rate(fluentbit_output_retries_failed_total[10m]) > 0
+      expr: rate(fluentbit_output_retries_failed_total{cluster_type="management_cluster"}[10m]) > 0
       for: 20m
```

Comment:
Can you just add a comment explaining why we only alert for MCs?

Reply:
<img src="https://media2.giphy.com/media/v1.Y2lkPWJkM2VhNTdldXBkYzg3bXBmcGc5cDR1OTltdzFhdzJmMXI3dHY3anh2Z2pmbWYyZSZlcD12MV9naWZzX3NlYXJjaCZjdD1n/xzMs8O1KQh17i/giphy.gif"/>